garage/tf/models/sequential.py

```diff
@@ -0,0 +1,48 @@
+"""
+Sequential Model.
+
+A model composed of one or more models which are connected sequential,
+according to the insertion order.
+"""
+from garage.tf.models.base import Model
+
+
+class Sequential(Model):
+    """
+    Sequential Model.
+
+    Args:
+        name: Variable scope of the Sequential model.
+        models: The models to be connected in sequential order.
+    """
+
+    def __init__(self, *models, name=None):
+        super().__init__(name)
+        self._models = models
+
+    def _build(self, input_var, name=None):
+        out = input_var
+        for model in self._models:
+            out = model.build(out, name=name)
```

Comment:
perhaps the names should be prefixed? e.g.

```python
for model in self._models:
    inner_name = '{}/{}'.format(name, type(model).__name__)
    out = model.build(out, name=inner_name)

return out
```

Reply:
Since the sub-model (e.g. `mlp`) is built in the parent model `Sequential`, it's already inheriting from the `Sequential` variable scope.

An example:
```python
policy = CategoricalConvPolicyWithModel(...)
for var in tf.global_variables():
    print(var)

'''
<tf.Variable 'CategoricalConvPolicy/Sequential/CNNModel/return_var:0' shape=() dtype=float32_ref>
<tf.Variable 'CategoricalConvPolicy/Sequential/MLPModel/return_var:0' shape=() dtype=float32_ref>
'''
```

Reply:
lgtm